cpp/oneapi/dal/data/undirected_adjacency_array_graph.hpp

```diff
@@ -23,6 +26,17 @@
 
 namespace oneapi::dal::preview {
 
+/// Class for a data management component responsible for representation of data
+/// in the graph format. The class is designed to minimize storage requirements 
+/// and offer good performance characteristics. Graph is stored in 0-based CSR
```

Comment:
and offer good performance characteristics. **The** graph is stored in **a** 0-based CSR